node_modules/oae-core/about/about.html

```diff
@@ -0,0 +1,48 @@
+<!-- CSS -->
+<link rel="stylesheet" type="text/css" href="css/about.css" />
+
+<div class="about-widget">
+    <div id="about-header"><!-- --></div>
```

Comment:
We usually suffix elements that will be filled with template content with `-container`